app/Resources/views/parts/blog_post/author.html.twig

```diff
@@ -0,0 +1,19 @@
+<div class="blog-post-author-image">
+    {{ ez_render_field(content, 'profile_picture', {
+        'parameters': {
+            'alias': 'blog_post_author'
+        }
+    }) }}
+</div>
+
+<div class="blog-post-author-name">
+    By <span>{{ ez_field_value(content, 'first_name') }} {{ ez_field_value(content, 'last_name') }}</span>
+</div>
+
+<div class="blog-post-author-bio">
+    {{ ez_render_field(content, 'bio') }}
+</div>
+
+<div class="blog-post-author-company">
+    {{ ez_render_field(content, 'company') }}
+</div>
```

Comment:
Social media icons are missing in this PR.
- github
- twitter
- linkedin
- facebook
- google plus

Important note: github account was not listed in the fields for the blog post content type. It should be added.